### DISCUSSION TITLE
Importation CSV ou XML

### DISCUSSION THREAD

**Discussion by ghost** [dbarzin/mercator#218] [Q&A]
Bonjour, j'ai découvert votre magnifique solution de cartographie. Cependant je ne trouve pas la possibilité d'importer des fichier XML ou CSV dedans. Est-il possible de le faire?

cordialement,

Erwan

**Comment by dbarzin** [comment]
Bonjour,

Il existe une [API](https://dbarzin.github.io/mercator/api/) avec des exemples d'utilisation dans différents langages.

Il est possible également d'importer des données avec la commande [load data](https://dev.mysql.com/doc/refman/8.0/en/load-data.html) ou avec [MySQL Workbench](https://dev.mysql.com/doc/workbench/en/wb-admin-export-import-table.html) ou encore [Heidi SQL](https://www.heidisql.com/help.php#import).

Il n'est pas prévu de le faire via l'application.

Didier
